data/korath/korath ships.txt

```diff
@@ -296,6 +314,151 @@ ship "Korath Raider" "Korath Raider (Crippled)"
 	turret "Korath Warder"
 
 
+ship "Korath Raider Mark XLII"
+	sprite "ship/raiderii"
+	thumbnail "thumbnail/raiderii"
+	attributes
+		category "Medium Warship"
+		"cost" 22395000
+		"shields" 27500
```

Comment:
```suggestion
		"shields" 29500
```

Let's give them an actual buff - not just a polish. With my suggested change here, that brings them up to a +9.2% shield capacity boost instead of the previous +1.8%